### DISCUSSION TITLE
Computer IDs

### DISCUSSION THREAD

**Discussion by Captain-Oppai** [cc-tweaked/CC-Tweaked#644] [Q&A] [answered]
After playing a modpack with updated mods and looking through some change logs I noticed the following change, "Computers and turtles now preserve their ID when broken."

I wanted to know if it's possible to disable this in any way, or if there are any alternative ways to stack turtles/computers.

**Comment by Lupus590** [comment, marked as answer]
Not currently, but I am considering making a PR which will add a way of doing this.
